components/Welcome.jsx

```diff
@@ -1,7 +1,23 @@
 import React from "react";
+import Image from "next/image";
 
 const Welcome = () => {
-  return <div>Welcome</div>;
+  return (
+    <div>
+      <h1 className="text-acm-black text-center font-bold xl:text-7xl l:text-6xl s:text-7xl text-5xl">
+        Welcome to
+      </h1>
+      <div className="bg-acm-blue rounded-3xl w-10/12 min-w-[300px] max-w-[450px] mx-auto my-3 px-6">
+        <Image
+          layout="responsive"
+          src="/acmucr.png"
+          alt="ACM @ UCR"
+          width={681}
```

Comment:
want to avoid hard coded values, should be based on the container alone, for now it is fine, before we merge let's setup the final containers that will contain this so its easier to understand how big we want the image

Reply:
I believe these values in width and height are simply the aspect ratio based off of documentation I read using Next Image components but I could be wrong. The div container that the image is in is responsible for changing the size of the image dynamically

Reply:
also tested it on a test container that using React grid components, just didn't add it to this branch since this branch was just working on the component. I have images above showing how it looks at some common screen sizes. Thanks!